tachyon/math/base/rational_field_unittest.cc

```diff
@@ -156,7 +156,7 @@ TEST_F(RationalFieldTest, MultiplicativeOperators) {
     R tmp = test.a;
     tmp *= test.b;
     EXPECT_EQ(tmp, test.mul);
-    tmp /= test.b;
+    ASSERT_TRUE(tmp /= test.b);
```

Comment:
I'm not sure if we can always assure the quotient to have value.

Reply:
There's only two concrete cases that `test` is in this test: `a = R(GF7(3), GF7(2))`, `b = R(GF7(2), GF7(5))` or `a = R(GF7(5), GF7(3))`, `b = R(GF7(1), GF7(2))`. In this case at least, the quotient will always have a value.